### PR TITLE
Feature send images

### PR DESCRIPTION
**Pull request by NickJocius**
1. Set up cloudinary account and added a new ImageUpload form, utilizing mui modal.
2. Used axios to make post request to cloudinary api and upload single or multiple files.
3. Used response data to create state and pass to Input.js to post message with additional attachments.
4. Displayed the new information in SenderBubble. 
5. Adjusted responsive styles for activeChat and its subcomponents.

img1
img2R

Closes issue #3

### CODE REVIEW

**Review by 1234tgk — COMMENTED** [review]
Great work on #3. The new feature works really well! 👍 

I noticed there is small avatar underneath each and every message. Please remove this small feature before requesting for the second review. Screenshot is included for reference.

Please let us know in the Slack channel if you have any questions or if the PR is ready for the second review!

image

**Review by FilipeGorodscy — APPROVED** [review]
Great job overall. There are only a few comments I left below that should be addressed before merging! 👍

**Pull request merged**
